### PR TITLE
Don't require multi_json and multi_xml.

### PR DESCRIPTION
**Pull request by dblock**
I think we need to add explicit configuration to this and tests that use different json and xml libraries.

### CODE REVIEW

**Comment by dblock** [comment]
@namusyaka How do we feel about this?

File: lib/grape/util/json.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+end
+
+module Grape
+  if Object.const_defined? :MultiJson
+    Json = ::MultiJson
+  else
+    Json = ::JSON
+    Json::ParseError = Json::ParserError
+  end
+end
```
Comment (namusyaka) [inline lib/grape/util/json.rb]: I'd like to declare `require 'json'` explicitly.

File: lib/grape/util/xml.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+rescue LoadError
+end
+
+module Grape
+  if Object.const_defined? :MultiXml
+    Xml = ::MultiXml
+  else
+    Xml = ::ActiveSupport::XmlMini
+    Xml::ParseError = StandardError
+  end
```
Comment (namusyaka) [inline lib/grape/util/xml.rb]: Likewise, I'd like to declare `require 'active_support/xml_mini'` explicitly. 
At the moment we don't declare `require 'active_support'` itself.

**Comment by namusyaka** [comment]
Sorry for the delay. I strongly agree with the purpose of this pull request.
Thank you!

File: lib/grape/util/json.rb (hunk 1 above)
Comment (dblock) [inline lib/grape/util/json.rb]: What do you mean exactly? That the user has to `require 'json'`?

File: lib/grape/util/xml.rb (hunk 2 above)
Comment (dblock) [inline lib/grape/util/xml.rb]: Or are you saying that I should add `require 'active_support/xml_mini'` here after `else`?

**Comment by dblock** [comment]
@namusyaka I made the change that requires you to require multi_* manually. If you care to take a look? I'm going to merge soon.

File: lib/grape/util/xml.rb (hunk 2 above)
Comment (namusyaka) [inline lib/grape/util/xml.rb]: Yes. Do I understand things correctly?

Comment (dblock) [inline lib/grape/util/xml.rb]: Ok, take a look at the PR as now, any issues? Thank you.

**Comment by namusyaka** [comment]
👍 👍 👍

**Comment by namusyaka** [comment]
Thank you for your great work! 🎉

**Pull request merged**